code/src/containers/forms/product-form/ProductForm.types.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Control,
+  FieldErrors,
+  UseFormRegister,
+  UseFormRegisterReturn
+} from "react-hook-form";
+
+import { AppInputProps } from "@/components/app-input/AppInput.types";
+
+import { Locale } from "@/context/i18n/I18nProvider";
+import { FullManagerProduct } from "@/types/product.types";
+
+export type ProductFormValues = {
+  status: boolean;
+  image: string;
+  quantity: number | undefined;
+  price: number | undefined;
+  category: number;
+  productTranslations: ProductTranslation[];
+};
+
+export type UpdateProductFormValues = {
+  status: boolean;
+  image: string;
+  quantity: number | undefined;
+  price: number | undefined;
+  tagIds: number[];
+  productTranslations: ProductTranslation[];
+};
```

Comment:
These two types have many in common. I suggest to create kinda parent local type and extend `ProductFormValues` and `UpdateProductFormValues` from it

Reply:
I deleted that type, it was unused